secure_message/services/party_service.py

```diff
@@ -10,18 +10,27 @@
 
 
 class PartyService:
+
+    __users_cache = dict()
```

Comment:
Generally the syntax for such operations is `my_var = {}`. This is both faster and generally more widely used. See this for an analysis of the performance https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/ (it is for Python 2.7.2 so a little outdated now)